packages/core/etc/components-core.api.md

```diff
@@ -347,7 +347,7 @@ export type SetMediaDeviceOptions = {
 };
 
 // @public (undocumented)
-export function setupChat(room: Room): {
+export function setupChat(room: Room, messageEncoder?: (message: string, timestamp: number) => Uint8Array, messageDecoder?: (message: Uint8Array) => ChatMessage): {

```

Comment:
I'd prefer to pack those two into an (optional) options object together

Reply:
That's perfect!
I've made the update